developer_setup.md

```diff
@@ -19,9 +19,8 @@
   sudo dnf -y install postgresql-devel postgresql-server # PostgreSQL Database server and to build 'pg' Gem
   sudo dnf -y install bzip2 libffi-devel readline-devel  # For rbenv install 2.2.0 (might not be needed with other Ruby setups)
   sudo dnf -y install libxml2-devel libxslt-devel patch  # For Nokogiri Gem
-  sudo dnf -y install gcc-c++                            # For event-machine Gem
   sudo dnf -y install sqlite-devel                       # For sqlite3 Gem
-  sudo dnf -y install nodejs npm                         # For ExecJS Gem and bower
+  sudo dnf -y install nodejs                             # For ExecJS Gem and bower
```

Comment:
@isimluk I've had [issues](https://bugzilla.redhat.com/show_bug.cgi?id=1339284#c0), when I tried installing both `npm` and `nodejs`on my fresh Fedora 24 installation, that's why I'm removing it in the PR.

I agree that the change could be confusing for users with older Fedora or CentOS, I just didn't find the right way to describe it.